Client/Frontend/Browser/Tabs/RemoteTabsPanel.swift

```diff
@@ -157,6 +170,8 @@ class RemoteTabsPanelClientAndTabsDataSource: NSObject, RemoteTabsPanelDataSourc
     func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
         guard let headerView = tableView.dequeueReusableHeaderFooterView(
             withIdentifier: SiteTableViewHeader.cellIdentifier) as? SiteTableViewHeader else { return nil }
+        headerView.theme = theme
```

Comment:
Why set the theme variable instead of calling `headerView.applyTheme(theme: theme)` ?